javascript/ql/src/semmle/javascript/NPM.qll

```diff
@@ -190,7 +190,7 @@ class BugTrackerInfo extends JSONValue {
   }
 
   /** Gets the bug tracker URL. */
-  string getURL() {
+  string getUrl() {
```

Comment:
There are two other similarly named `getURL` methods in the same file.  
Could you rename those while you are at it? 

A quick grep through the repo suggests that no other file contain the same mistake.

Reply:
Oh good catch. Grep was indeed how I found that one, apparently I just can't read.